fleet/lib/loki-pre/overlays/antu/externalsecret-loki-s3.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: external-secrets.io/v1beta1
```

Comment:
this file is identical to the previous on kueyen, this bundle doesn't need a overlay, please remove it.